### ISSUE TITLE
Node Plugin Causing RVM Issues

### ISSUE THREAD

**Issue by paulo-rezende-ifood** [Bash-it/bash-it#330]
This line on node Plugin is placing a reference to node bin directory at the beggining of PATH variable:
https://github.com/revans/bash-it/blob/master/plugins/available/node.plugin.bash#L4

First of all, i don't think this line is correct because it's using relative paths on PATH variable, but i'm not a Node.js expert nor bash expert. so i'm not sure if it's valid or not.

But the main point is that placing it on the begging of PATH variable is causing RVM to display this warming:

``` sh
Warning! PATH is not properly set up, '/usr/local/rvm/gems/ruby-1.9.3-p547/bin' is not at first place,
         usually this is caused by shell initialization files - check them for 'PATH=...' entries,
         it might also help to re-add RVM to your dotfiles: 'rvm get stable --auto-dotfiles',
         to fix temporarily in this shell session run: 'rvm use ruby-1.9.3-p547'.
```

And even if you try to execute the recommended command, it will not be solved because the node plugin will continue to place the value on the begging of PATH.

The obvious solution is to invert the order of the PATH:

``` sh
#from
export PATH=./node_modules/.bin:$PATH
#to
export PATH=$PATH:./node_modules/.bin
```

But since i'm not sure of the validity of the relative path itself i'm not submiting a PR directly, if you think this is the best solution and want me to send the PR just let me know.

I found this StackOverflow question that may clarify a bit more the problem: http://stackoverflow.com/questions/9679932/how-to-use-package-installed-locally-in-node-modules

Thanks.

**Comment by cornfeedhobo** [comment]
This was solved in #1404

@nwinkler This can be closed.

**Comment by nwinkler** [comment]
Thanks for the update, closing this now!
